genie-web/src/test/groovy/com/netflix/genie/web/tasks/job/JobCompletionServiceSpec.groovy

```diff
@@ -121,12 +124,8 @@ class JobCompletionServiceSpec extends Specification {
                 { throw new GenieServerException("null") } >>
                 { throw new GenieServerException("null") } >>
                 { throw new GenieServerException("null") }
-        completionTimerId.withTags(MetricsUtils.newFailureTagsMapForException(new GenieServerException("null")))
```

Comment:
If you are expecting this to be called on your timer, you should prefix it with `1 *`

Reply:
This line was removed in order to fix the test